test/e2e-upgrade-tests.sh

```diff
@@ -44,6 +44,8 @@ initialize "$@" --skip-istio-addon
 
 TIMEOUT=${TIMEOUT:-60m}
 
+export GO_TEST_VERBOSITY=standard-verbose
+
```

Comment:
What is this for? I don't see any usage of this?

```suggestion
```

Reply:
By default go_test_e2e now will not have any output when the test succeeds. This is useful in integration test, but the upgrade test does not output lots of logs, and I think it is nice to see the log since we are actually turning off the event delivery failure now. Setting this env var will enable the log like before.

Without this we will only see sth like: 
```
upgrade-test: PASS
```

Reply:
OK. So, let's write that as, to enable configurability:

```suggestion
export GO_TEST_VERBOSITY="${GO_TEST_VERBOSITY:-standard-verbose}"

```

Reply:
Changed as suggested.